### ISSUE TITLE
Error calling server command

### ISSUE THREAD

**Issue by bookofportals** [evennia/evennia#1635]
#### Brief summary of issue / Description of requested feature:

An exception is thrown whenever I call `@server`.

#### Steps to reproduce the issue / Reasons for adding feature:

1. Log in normally
2. Enter the command `@server`

#### Error output / Expected result of feature
```
Traceback (most recent call last):
  File "/usr/src/evennia/evennia/commands/cmdhandler.py", line 591, in _run_command
    ret = cmd.func()
  File "/usr/src/evennia/evennia/commands/default/system.py", line 769, in func
    rmem = float(os.popen('ps -p %d -o %s | tail -1' % (pid, "rss")).read()) / 1000.0  # resident 
memory
ValueError: could not convert string to float: 

An untrapped error occurred.
(Traceback was logged 2018-07-17 01:10:38-0000).
```

#### Extra information, such as Evennia revision/repo/branch, operating system and ideas for how to solve / implement:
I'm running this in a docker container on Mac OS, using the `evennia/evennia` image as a base. The same error occurs for `@server/mem` and `@serverprocess`.

**Comment by bookofportals** [comment]
@Griatch Unless I'm mistaken, your commit resolves #1636, not this issue.

**Comment by Griatch** [comment]
@bookofportals  Oops, correct. Mis-wrote in the commit, thanks.
